manifests/init.pp

```diff
@@ -242,6 +244,7 @@
   String $datadog_site = $datadog_agent::params::datadog_site,
   String $host = '',
   String $api_key = 'your_API_key',
+  String $agent_flavor = $datadog_agent::params::package_name,
```

Comment:
Another nit: we could have
```suggestion
  Enum['datadog-agent', 'datadog-iot-agent'] $agent_flavor = $datadog_agent::params::package_name,
```
to prevent silly mistyping errors.

Reply:
Actually, on Windows the package name is `Datadog Agent` so this won't work. Let me check if we can use `$datadog_agent::params::package_name` as one of the values of the enum.

Reply:
We can always add `Datadog Agent` too I guess?

Reply:
I've done that.